typescript-operators/main.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+/* eslint-disable no-unused-vars */
+const v1 = { value: 'Something' };
+const v2 = '';
+const v3 = undefined;
+const v4 = [3, 5, 7, 9];
+const v5 = {
+  name: 'Carl',
+  details: {
+    age: 82,
+  },
+};
+// && Logical AND operator
+console.log(v1 && 'v1 is truthy', v2 && 'v2 is truthy');
+// || Logical Or Operator
```

Comment:
This one is still _always_ logging to the console. You should only log `'v1 is truthy'` if v1 is truthy, and `'v2 is truthy'` if v2 is truthy. The way the logical and is written is correct, but your console log is in the wrong place.